src/main/java/org/embulk/util/aws/credentials/AwsCredentials.java

```diff
@@ -57,7 +57,7 @@ private static AWSCredentialsProvider getAWSCredentialsProvider(String prefix, A
         String profileNameOption = prefix + "profile_name";
         String accessKeyIdOption = prefix + "access_key_id";
         String secretAccessKeyOption = prefix + "secret_access_key";
-        String accountIdOption = prefix + "account_id";
+        String awsAccountIdOption = prefix + "aws_account_id";
```

Comment:
Ah, sorry, one point. Do you know what can come to `prefix`? I was afraid that `prefix` can be `aws_`. Then it can be `aws_aws_account_id`...

Reply:
Yeah, `aws_` would come.

https://github.com/embulk/embulk-util-aws-credentials/pull/8/files#diff-175aa51d5ebfd84c1e5e273722fdfa278b5c47fb46225f8cd7b25b9381537e9aR39-R41

Any reason why just `account_id` was not good?

Reply:
Sorry, i don't notice we have a aws_ prefix configuration. I just change back to account_id and add these configuration  to AwsCredentialsTaskWithPrefix.